### PR TITLE
[DX-930] Update endpoint-designer.md

### PR DESCRIPTION
**Pull request by letzya**
- fix bug in a hyper link (you can't have BOLD MD syntax in a link
- improve content of the ignore case
- improve content of the regexp $ sign

Preview 
https://deploy-preview-3832--tyk-docs.netlify.app/docs/nightly/advanced-configuration/transform-traffic/endpoint-designer/

### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

---

### Preview Link

<br>

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labelled** the PR

### CODE REVIEW

**Comment by letzya** [comment]
@ilijabojanovic where do i find the setting to enforce ignore case on the api level, not the endpoint level?

File: tyk-docs/content/advanced-configuration/transform-traffic/endpoint-designer.md [hunk 1]
```diff
@@ -25,18 +25,24 @@ By default, importing an API using Swagger/OpenAPI or API Blueprint JSON definit
...
 - **Plugin**: A path can belong to multiple plug-ins, these plug-ins define the behaviour you want to impose on the matched request.
 
 {{< note success >}}
 **Note**  
 
-When using Regular Expressions with the following plugins (Mock Response, {{<fn>}}Blocklist{{</fn>}} and {{<fn>}}Allowlist{{</fn>}}) you need to add `$` to the end of your URL. This prevents anything following the endpoint being mocked as well. For example, adding `/mock` also means `/mock/somepath` can also be mocked. Using `/mock$` prevents `/somepath` being added and mocked to your endpoint.
+**Endpoint parsing**
+With the following plugins (Mock Response, {{<fn>}}Blocklist{{</fn>}} and {{<fn>}}Allowlist{{</fn>}}) adding a `$` at the end of your URL. This ensures that regular expression matching avoids endpoints with characters following the specified endpoint.
```
Comment (dcs3spp) [inline tyk-docs/content/advanced-configuration/transform-traffic/endpoint-designer.md]:
```suggestion
The following plugins (Mock Response, {{<fn>}}Blocklist{{</fn>}} and {{<fn>}}Allowlist{{</fn>}}) require a `$` added at the end of your URL. This ensures that regular expression matching is exact, avoiding endpoints with characters following the specified endpoint.
```

File: tyk-docs/content/advanced-configuration/transform-traffic/endpoint-designer.md [hunk 2]
```diff
@@ -25,18 +25,24 @@ By default, importing an API using Swagger/OpenAPI or API Blueprint JSON definit
...
 
 {{< note success >}}
 **Note**  
 
-When using Regular Expressions with the following plugins (Mock Response, {{<fn>}}Blocklist{{</fn>}} and {{<fn>}}Allowlist{{</fn>}}) you need to add `$` to the end of your URL. This prevents anything following the endpoint being mocked as well. For example, adding `/mock` also means `/mock/somepath` can also be mocked. Using `/mock$` prevents `/somepath` being added and mocked to your endpoint.
+**Endpoint parsing**
+The following plugins (Mock Response, {{<fn>}}Blocklist{{</fn>}} and {{<fn>}}Allowlist{{</fn>}}) require a `$` added at the end of your URL. This ensures that regular expression matching is exact, avoiding endpoints with characters following the specified endpoint.
+<br/>
+Here's an example to illustrate its impact: When you define an `/anything` endpoint, it also implies a potential match for `/anything/somepath`. However, employing `/anything$` in your definition specifically prevents the matching of `/somepath`. This distinction is interpreted as follows:
+- In an allowlist setup, only `/anything` is permitted, and `/anything/somepath` remains blocked unless explicitly added.
```
Comment (dcs3spp) [inline tyk-docs/content/advanced-configuration/transform-traffic/endpoint-designer.md]:
```suggestion
- In an Allowlist setup, only `/anything` is permitted, and `/anything/somepath` remains blocked unless explicitly added.
```

File: tyk-docs/content/advanced-configuration/transform-traffic/endpoint-designer.md [hunk 3]
```diff
@@ -59,14 +65,14 @@ Accessing a path which has **not** been allowed:
 }
 ```
 
-#### Case Sensitivity
+#### Case Sensitivity of Allowlist
 
-By default the {{<fn>}}Allowlist{{</fn>}} endpoint plugin is case-sensitive, so for example if `getuser` is allowed, `getUser` and `GetUser` will not be allowed. If you select the **Ignore Case** option from the {{<fn>}}Allowlist{{</fn>}} plugin settings, `getUser`, `GetUser` and `getuser` will all be allowed in the above example.
+By default the {{<fn>}}Allowlist{{</fn>}} endpoint plugin is case-sensitive, so for example if `getuser` is allowed, while `getUser` and `GetUser` are not. If you select the **Ignore Case** option in the {{<fn>}}Allowlist{{</fn>}} plugin settings, all three options will be allowed.
```
Comment (dcs3spp) [inline tyk-docs/content/advanced-configuration/transform-traffic/endpoint-designer.md]:
```suggestion
By default the {{<fn>}}Allowlist{{</fn>}} endpoint plugin is case-sensitive, so for example `getuser` is allowed, while `getUser` and `GetUser` are not. If you select the **Ignore Case** option in the {{<fn>}}Allowlist{{</fn>}} plugin settings, all three options will be allowed.
```

**Review by dcs3spp — APPROVED** [review]
PR looks good to me. I committed some minor suggestions to the PR. Once peer reviewed will merge and release

File: tyk-docs/content/advanced-configuration/transform-traffic/endpoint-designer.md [hunk 4]
```diff
@@ -25,18 +25,24 @@ By default, importing an API using Swagger/OpenAPI or API Blueprint JSON definit
...
+- **Relative Path**: The relative path to the target. For example, if your API is listening on an `/api` listen path, and you want to match the `/api/get` URL, in the Endpoint Designer you should match for the `/get` endpoint. It is important to exclude aberrant slashes (`/`) from your path matching, as otherwise the gateway may not match the path correctly. A path can contain wild cards, such as `{id}`, the actual value in the wildcard is not used (it is translated into a regex), however, it is useful to make the path more human-readable when editing.
 - **Plugin**: A path can belong to multiple plug-ins, these plug-ins define the behaviour you want to impose on the matched request.
 
 {{< note success >}}
 **Note**  
 
-When using Regular Expressions with the following plugins (Mock Response, {{<fn>}}Blocklist{{</fn>}} and {{<fn>}}Allowlist{{</fn>}}) you need to add `$` to the end of your URL. This prevents anything following the endpoint being mocked as well. For example, adding `/mock` also means `/mock/somepath` can also be mocked. Using `/mock$` prevents `/somepath` being added and mocked to your endpoint.
+**Endpoint parsing**
```
Comment (letzya) [inline tyk-docs/content/advanced-configuration/transform-traffic/endpoint-designer.md:39]: @ilijabojanovic can you please review this?

File: tyk-docs/content/advanced-configuration/transform-traffic/endpoint-designer.md [hunk 5]
```diff
@@ -59,14 +65,14 @@ Accessing a path which has **not** been allowed:
 }
 ```
 
-#### Case Sensitivity
+#### Case Sensitivity of Allowlist
```
Comment (letzya) [inline tyk-docs/content/advanced-configuration/transform-traffic/endpoint-designer.md:68]: @ilijabojanovic  here

File: tyk-docs/content/advanced-configuration/transform-traffic/endpoint-designer.md [hunk 6]
```diff
@@ -59,14 +65,14 @@ Accessing a path which has **not** been allowed:
...
 
-By default the {{<fn>}}Allowlist{{</fn>}} endpoint plugin is case-sensitive, so for example if `getuser` is allowed, `getUser` and `GetUser` will not be allowed. If you select the **Ignore Case** option from the {{<fn>}}Allowlist{{</fn>}} plugin settings, `getUser`, `GetUser` and `getuser` will all be allowed in the above example.
+By default the {{<fn>}}Allowlist{{</fn>}} endpoint plugin is case-sensitive, so for example `getuser` is allowed, while `getUser` and `GetUser` are not. If you select the **Ignore Case** option in the {{<fn>}}Allowlist{{</fn>}} plugin settings, all three options will be allowed.
 
 {{< note success >}}
 **Note**  
 
-You can also use `ignore_endpoint_case` at a ["global" Tyk level]({{< ref "tyk-oss-gateway/configuration#ignore_endpoint_case" >}}) in your `tyk.conf` file and at an individual API level. Those settings will ovverride this setting. This is new for v2.9.4.
+You can also set a global ignore case on the API level or across [the gateway]({{< ref "tyk-oss-gateway/configuration#ignore_endpoint_case" >}}) in `tyk.conf`. These global settings will override this endpoint-level setting. (Added in v2.9.4).
```
Comment (letzya) [inline tyk-docs/content/advanced-configuration/transform-traffic/endpoint-designer.md:75]: @ilijabojanovic here

**Review by ilijabojanovic — APPROVED** [review]
LGTM

**Pull request merged**